client/script.js

```diff
@@ -0,0 +1,532 @@
+const root = document.getElementById('root');
+
+
+// CREATING NAV BAR
+const nav = ['Travel updates', 'Reviews', 'About', 'Contact'];
+
+function createNav(nav) {
+    const navBar = document.createElement('nav');
+    navBar.setAttribute('class', 'nav');
+
+    const ul = document.createElement('ul');
+    ul.setAttribute('class', 'nav__container');
+    navBar.appendChild(ul);
+
+    nav.forEach(element => {
+        const li = document.createElement('li');
+        li.setAttribute('class', 'nav__item');
+        const anchor = document.createElement('a');
+        anchor.setAttribute('href', '#/');
+        anchor.setAttribute('class', 'nav__link');
+        anchor.textContent = element;
+
+        ul.appendChild(li);
+        li.appendChild(anchor);
+
+    })
+
+    return navBar;
+}
+
+function renderNavBar(nav) {
+    const domNavBar = createNav(nav);
+    root.appendChild(domNavBar);
+    createNav(nav);
+
+}
+
+
+// CREATING THE ADD BUTTON
+
+function createAddButton() {
+    const div = document.createElement('div');
+    div.setAttribute('class', 'add__container');
+    const button = document.createElement('button');
+    button.setAttribute('type', 'button');
+    button.setAttribute('class', 'button open-modal');
+    button.textContent = '+ Add Article';
+    button.addEventListener('click', openModal)
+    div.appendChild(button);
+
+    return div;
+}
+
+function renderAddButton() {
+    const domButton = createAddButton();
+    root.appendChild(domButton);
+    createAddButton();
+}
+
+
+// TAKING DATA FROM SERVER
+fetch('http://localhost:3000/articles')
+    .then(
+        function(response) {
+            if (response.status !== 200) {
+
+                console.log('Looks like there was a problem. Status Code: ' +
+                    response.status);
+                return;
+            }
+
+            // Examine the text in the response
+            response.json()
+                .then(data => {
+                    console.log(data)
+                    window.onhashchange = locationHashChange(data);
+                });
+        }
+    )
+    .catch(function(err) {
+        console.log('Fetch Error :-S', err);
+    });
+
+
+// CREATE ALL ARTICLES FROM MAIN PAGE 
+function createArticle(articles) {
+    const domArticle = document.createElement('article');
+    console.log(articles)
+
+    articles.forEach(element => {
+        const articleDiv = document.createElement('div');
+        articleDiv.setAttribute('id', 'article' + element.id);
+        const domTitle = document.createElement('h2');
+        domTitle.textContent = element.title;
+        domTitle.setAttribute('class', 'title');
+
+        const domUl = document.createElement('ul');
+        domUl.setAttribute('class', 'info__container');
+
+        const domLi1 = document.createElement('li');

```

Comment:
domLi1, domLi2., classLi ar putea fi redenumite mai sugestiv, sa intelegem mai usor ce se retine in fiecare. 😃